web/main.go

```diff
@@ -0,0 +1,344 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"github.com/labstack/echo"
+	"html/template"
+	"io"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type HomeFilters struct {
+	State    string
+	Year     string
+	City     string
+	Position string
+}
+
+type CandidateTag struct {
+	Tag         string
+	Description string
+}
+
+type Candidate struct {
+	Name                   string
+	Email                  string
+	Party                  string
+	NumberOfTerms          int
+	CandidatureNumber      string
+	Position               string
+	City                   string
+	ImageURL               string
+	TransparencyPercentage int
+	Biography              string
+	SocialLinks            *[]SocialLink
```

Comment:
Acho que tu quiz dizer: 

```suggestion
	SocialLinks            []*SocialLink
```

Porque se vc faz *[]SocialLink na prática tu tá retornando um ponteiro pra outro ponteiro

Reply:
Confesso que não sei usar ponteiros (faz tempo que não uso). Eu vi uns exemplos e funcionou, hehe

Reply:
kkkkkkkkkkkk
normal, foi a mesma coisa quando comecei a usar Go, tava com a cabeça no kotlin ai me enrolava um pouco